public/app/auth.js

```
@@ -0,0 +1,230 @@
+  // ─── Init ────────────────────────────────────────────────────────────────────
+  async function init() {
+    const session = _loadSession();
+    if (session && session.token) {
+      // Try to restore session
+      try {
+        const me = await _fetchMe(session.token);
+        if (me && me.username) {
+          _currentUser = me;
+
+          // Re-activate license from stored token
+          if (window.LicenseManager && typeof window.LicenseManager.activate === 'function') {
```

ESLint

[error] 'window' is not defined.

(no-undef)